docs/en_US/Release.md

```diff
@@ -153,7 +153,7 @@
   * Run trial jobs on the GPU running non-NNI jobs
 * Kubeflow v1beta2 operator
   * Support Kubeflow TFJob/PyTorchJob v1beta2
-* [General NAS programming interface](AdvancedFeature/GeneralNasInterfaces.md)
+* *General NAS programming interface (deprecated)*
```

Comment:
no need to change this part

Reply:
work around in 31788b692de591a6557579ce203b8838c32b424c